cognite_toolkit/_cdf_tk/loaders/_resource_loaders/auth_loaders.py

```diff
@@ -315,23 +317,16 @@ def dump_resource(self, resource: Group, local: dict[str, Any] | None = None) ->
         # When you dump a CDF Group, all the referenced resources should be available in CDF.
         return self._substitute_scope_ids(dumped, is_dry_run=False, reverse=True)
 
-    def _upsert(self, items: Sequence[GroupWrite]) -> GroupList:
+    def create(self, items: Sequence[GroupWrite]) -> GroupList:
         if len(items) == 0:
             return GroupList([])
         # We MUST retrieve all the old groups BEFORE we add the new, if not the new will be deleted
         old_groups = self.client.iam.groups.list(all=True)
-        old_group_by_names = {g.name: g for g in old_groups.as_write()}
-        to_create = []
-        for item in items:
-            if (old := old_group_by_names.get(item.name)) and old == item:
-                # Skip unchanged groups
-                continue
-            to_create.append(item)
```

Comment:
This could cause issues. Since we no longer support update, I think we just create new groups no matter what.